src/pages/settings/templates-and-reminders/TemplatesAndReminders.tsx

```diff
@@ -371,7 +373,10 @@ export function TemplatesAndReminders() {
                 </option>
               ))}
 
-            <option value="payment_failed">{t('payment_failed')}</option>
```

Comment:
@beganovich @turbo124 The `payment_failed` template has been removed as a static option since it's now returned by `statics.templates`.

Reply:
I'm not sure I follow. Why would the template be removed?

Reply:
@turbo124 When I initially added payment_failed to the dropdown options, it was a static option not coming from `statics.templates`. Now that I see `payment_failed` is included in the `statics.templates` response, I removed it from static options to avoid duplication. The template remains available in the dropdown through the query response, while other static options like `custom1` and `custom2` (which aren't in the query response) stay as static entries. Screenshot:

![Screenshot 2024-11-24 at 23 11 44](https://github.com/user-attachments/assets/f2ee764f-9ab7-4c67-bc91-2999ccdbf15c)

Reply:
@turbo124 can you please confirm this is OK?

Reply:
I believe now that it is being returned by the statics route, it is no longer required to be forced in the UI.